doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -418,7 +418,7 @@ Bluetooth Fast Pair samples
 Bluetooth Mesh samples
 ----------------------
 
-* Added support for the :ref:`zephyr:nrf54l15dk_nrf54l15` board in the following samples:
+* Added support for the :ref:`zephyr:nrf54l15dk_nrf54l15` board and removed support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board in the following samples:
```

Comment:
Will the PDK reference exist in the 2.8.0 release?

Reply:
If not, there is not point merging this link now if we have to remove it later. If we don't know, it's fine by me to keep it like this.

Reply:
@mia-ko: Do you know if this reference will be there?

Reply:
I have heard rumors that it won't be removed, so the reference should be working. I would still recommend we don't use it. Just to be on the safe side :)